CMakeLists.txt

```diff
@@ -39,6 +39,7 @@ option(MICM_ENABLE_OPENACC "Build with OpenACC Support" OFF)
 option(MICM_ENABLE_LLVM "Build with LLVM support for JIT-compiling" OFF)
 option(MICM_ENABLE_TESTS "Build the tests" ON)
 option(MICM_ENABLE_EXAMPLES "Build the examples" ON)
+option(MICM_ENABLE_PROFILE "Build the instrumentation" ON)
```

Comment:
```suggestion
option(MICM_ENABLE_PROFILE "Build the instrumentation. MICM_ENABLE_JSON must also be enabled " ON)
```

Reply:
The function `WriteProfile(const ProfileResult& result)` in `micm/profiler/instrumentation.hpp` handles the JSON format so I don't think that JSON is required for profiling by itself. What do you think?